dashboard/src/app/index.module.ts

```diff
@@ -82,12 +82,12 @@ function keycloakLoad(keycloakSettings: any) {
     script.addEventListener('load', resolve);
     script.addEventListener('error', () => {
       return cheBranding.ready.then(() => {
-        reject(`<div  class="error-header"><span>Certificate Error</span><a href="/"><i class="fa fa-times"></i></a></div>
- <div class="error-body"><p>Your Che host may be signed with a self-signed certificate. To resolve this issue, try these possible solutions:</p>
- <p>1.) Import CA certificate info into your browser. You can find instructions on how to do this in you 
+        reject(`<div class="header"><i class="fa fa-warning"></i><p>Certificate Error</p></div>
+ <div class="body"><p>Your Eclipse Che host may be signed with a self-signed certificate. To resolve this issue, try this possible solution:</p>
```

Comment:
"Your Eclipse Che server may be using a self-signed certificate. To resolve this issue, try to import the servers CA certificate into your browser, as described <a href= ....>here</>. After importing the certificate, refresh your browser.

Reply:
And don't forget about branding. CRW must have something like `CodeReady Workspaces server` instead of Eclipse Che

Reply:
done